### ISSUE TITLE
[Hacktoberfest] Contribution guide at OpenGenus/Cosmos 🌱

### ISSUE THREAD

**Issue by AdiChat** [OpenGenus/cosmos#5907]
You may use this guide to make quality contributions at **Hacktoberfest 2020**: [Hacktoberfest 2020 guide](https://iq.opengenus.org/hacktoberfest-2020/)

---

To begin contributing, you shall get a quick overview of how Cosmos is used today. **Cosmos** is one of the most extensive code database and is used as:

* A datastore for various projects (such as [Quark](https://github.com/OpenGenus/quark)) for purposes such as offline searching
* Training ML models for applications such as identifying programming language
* Metadata for projects such as [OpenGenus IQ](https://iq.opengenus.org/), [Falcon](https://github.com/OpenGenus/falcon) and several others
* Offline Learning resource by several contributors

To get started on your journey of making Cosmos and related OpenGenus projects better, follow the following steps:

* Fork Cosmos to get a local copy as your workspace
* Learn how to make a perfect pull request ([guide](https://iq.opengenus.org/create-a-pull-request-at-github/))
* Make a basic contribution like improving existing implementations, simple bugs (like #5891) and more to get familiar with the flow
* Make quality contributions ([more details](https://iq.opengenus.org/hacktoberfest-2020/)). This helps you in multiple angles

Some quality contributions:

* Implement new algorithms in language of your choice (Having strong implementation skills is useful)
* Write at OpenGenus IQ on a topic you are confident in or prepare questions on a given topic (Ability to explain ideas is important)
* Write at OpenGenus IQ on a new topic in the domain you are interested in (Approaching new topics correctly is the key to invention)
* Develop features for sister projects of Cosmos (challenging but interesting)

We, strongly, suggest you to you us as an author (apply to be [an author](https://discourse.opengenus.org/t/become-a-writer-at-opengenus-iq/843) or [an Intern](http://internship.opengenus.org)) and write on topics of your choice. We add this as a metadata to Cosmos (for example: #5830) which makes it complete.

This helps you grow intellectually 🧠 and help you gain good opportunities. Search and ask our Interns how they have grown over the years.

---

**Have a doubt or want us to review your contribution?** Ask us down 👇 and we will help you.

**Comment by UtkarshChaurasia** [comment]
Want to add implementation of recommendation system in AI section.

**Comment by AdiChat** [comment]
> Want to add implementation of recommendation system in AI section.

@UtkarshChaurasia : Sounds good, do open a new issue mentioning the task ("_implementation of recommendation system_") and submit your pull request. We will review it 😃

**Comment by Mayankjha997** [comment]
hey ! 
I would like  to add algorithm of rock paper and scissor in c language

**Comment by gsnsg** [comment]
Hi, I would like to add the Topological Sort algorithm in swift

**Comment by AdiChat** [comment]
@Mayankjha997 + @gsnsg Sure, you may go ahead and open a pull request for review.

To work on the explanation of the algorithms, you may let me know your username as in [our forum](https://discourse.opengenus.org) so that I can share an invite.

**Comment by gsnsg** [comment]
@AdiChat Here is the PR for the algorithm #5936

**Comment by Mayankjha997** [comment]
Please review my code

**Comment by Mayankjha997** [comment]
@AdiChat  here is the PR ,please review it #5947

**Comment by Mayankjha997** [comment]
@AdiChat sir please review my code

**Comment by AdiChat** [comment]
@Mayankjha997 Your pull request looks good and has been merged 😄

**Comment by gsnsg** [comment]
@AdiChat Please review my PR #5936

**Comment by NirmalSilwal** [comment]
Hello @AdiChat,  as discussed in issue #3732 I have added code for Convolutional Neural Network. Please **review my PR** **#5976** .

**Comment by syeedameen** [comment]
@AdiChat Greetings of the day Sir , I am a beginner in open source .
I would like to work on this.
And I am also taking part in Hacktoberfest , So can I contribute to it in October.

**Comment by gsnsg** [comment]
Hey @AdiChat please review my PR #5937

**Comment by piyush-cosmo** [comment]
Hey @AdiChat , I added a new optimization technique called policy gradients. The use of OpenAI gym and how to work with CartPole environment(how to balance the pole on a moving cart with policy gradients).

Please review my PR #6003

**Comment by Rutuja07** [comment]
Hey I would like to add algorithms in dart language.

**Comment by VkRan** [comment]
Hey I've added the hover effect in css in my commit. I hope you will accept it.

**Comment by brat-blip** [comment]
hey @AdiChat  i am a beginner in open source a issue(#6092) is assigned to me and i already make the pull request (#6111)so please review as soon as possible and merged it

**Comment by Succworth** [comment]
hey @AdiChat I would like to add an algorithm for hangman in Java
